### PR TITLE
implement REST endpoint to force broadcast of pending bls changes

### PR DESCRIPTION
**Pull request by potuz**
This PR implements a REST endpoint to force the beacon to broadcast all pending BLS changes in the operation pool

### CODE REVIEW

File: beacon-chain/rpc/apimiddleware/structs.go [hunk 1]
```diff
@@ -692,6 +692,10 @@ type BLSToExecutionChangeJson struct {
 	ToExecutionAddress string `json:"to_execution_address" hex:"true"`
 }
 
+type BroadcastBLSToExecutionChangesRequest struct {
+	Data string `json:"data"`
+}
```
Comment (rkapka) [inline beacon-chain/rpc/apimiddleware/structs.go:697]: Maybe let's create an empty struct that can be reused whenever we want to POST without data.
```
type EmptyRequest struct {}
```

**Review by terencechain — CHANGES_REQUESTED** [review]
Codewise, this looks good. My only ask:
Can anyone write up a short rationale and user guide on recommendations "how to use this" before we merge this?

**Comment by james-prysm** [comment]
Theres some suggestions from deepsource on this.

File: beacon-chain/rpc/eth/beacon/pool.go [hunk 2]
```diff
@@ -307,6 +308,25 @@ func (bs *Server) SubmitVoluntaryExit(ctx context.Context, req *ethpbv1.SignedVo
 	return &emptypb.Empty{}, nil
 }
 
+// BroadcastBLSToExecutionChanges broadcasts all existing BLS changes in the
+// operations pool.
```
Comment (james-prysm) [inline beacon-chain/rpc/eth/beacon/pool.go:312]: you responded on slack but could you add some use-case comments here as the reason for having this endpoint is less obvious.

**Comment by potuz** [comment]
Close as not needed

**Pull request closed without merging**